packages/hadron-app-registry/src/app-registry.ts

```diff
@@ -387,4 +375,9 @@ export class AppRegistry {
   }
 }
 
+/**
+ * Create a global app registry and prevent modification.
+ */
+export const globalAppRegistry = Object.freeze(new AppRegistry());
```

Comment:
Moved for convenience, doesn't change the behavior, will make some other upcoming changes we are planning for app registry easier